portal/exercise_diet/views.py

```diff
@@ -18,39 +21,63 @@ def get_asset(uuid):
 
 def get_tag_data(anyTags):
     url = (
-        "{LR_ORIGIN}/c/portal/truenth/asset/query?anyTags={anyTags}&sort=true&sortType=DESC".format(
+        "{LR_ORIGIN}/c/portal/truenth/asset/query?anyTags={anyTags}&sort=true"
+        "&sortType=DESC".format(
             LR_ORIGIN=current_app.config.get("LR_ORIGIN"), anyTags=anyTags))
```

Comment:
Might be better to lookup the value directly (`current_app.config.get["LR_ORIGIN"]`), instead of with `get()`, which will fail silently.